src/script/service/CoreSingleton.ts

```diff
@@ -38,6 +47,12 @@ export class Core extends Account {
       },
       enableMLS: Config.getConfig().FEATURE.ENABLE_MLS,
       nbPrekeys: 100,
+      /*
+       * When in an electron context, the window.secretsCrypto will be populated by the renderer process.
+       * We then give those crypto primitives to the core that will use them when encrypting MLS secrets.
+       * When in an browser context, then this secretsCrypto will be undefined and the core will then use it's internal encryption system
+       */
+      secretsCrypto: window.secretsCrypto,
```

Comment:
the failing test is a type issue here

Reply:
Yes, needs the webpackages update https://github.com/wireapp/wire-web-packages/pull/4302/files

Reply:
ah duhh sorry